packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -71,7 +71,8 @@ class CreateCommand extends FlutterCommand with CreateBase {
       allowed: <String>['objc', 'swift'],
       help:
           '(deprecated) The language to use for iOS-specific code, either Swift (recommended) or Objective-C (legacy). '
-          'Only supported for "--template=plugin".',
+          'Only supported for "--template=plugin". '
+          'This command will be removed in a future version of Flutter. Swift is now the default language for iOS plugins.',
```

Comment:
This PR makes it unsupported for any template, it's not about plugins, it's just that we've removed support for all templates before now except for plugins, and plugins is now being removed.
```
          '(deprecated) This command will be removed in a future version of Flutter.'
```

Reply:
Got it! Updated Help Message